apps/careers/tf/careers.tf

```diff
@@ -0,0 +1,283 @@
+provider "aws" {
+  region = "${var.region}"
+}
+
+terraform {
+  backend "s3" {
+    bucket = "mozilla-careers-s3-provisioning-tf-state"
+    key    = "tf-state"
+    region = "us-west-2"
+  }
+}
+
+resource "aws_iam_user" "careers" {
```

Comment:
unless it needs to be public, we store all IAM TF [here](https://github.com/mozmeao/infra-private/tree/master/aws/iam_policies)

Reply:
Removed from here and opened a PR against private repo. https://github.com/mozmeao/infra-private/pull/153

Thanks!